apps/server/src/providerApiClients/mx.ts

```diff
@@ -1,11 +1,9 @@
 import providerCredentials from '../providerCredentials'
 import { Configuration, MxPlatformApiFactory } from './mxClient'
-export * from './mxClient/api'
-export * from './mxClient/configuration'
 
 export const BASE_PATH = 'https://api.mx.com'.replace(/\/+$/, '')
 
-export const MxProdApiClient = MxPlatformApiFactory(
+export const MxProdApiClient: any = MxPlatformApiFactory(
```

Comment:
Linter doesn't like it without a type. It throws a strange error that I've never seen before, but this fixes it. The actual Type is very messy, and unfortunately, the mx-platform-node library doesn't cleanly define a type.